### DISCUSSION TITLE
Themes usage

### DISCUSSION THREAD

**Discussion by Psijic** [mapsforge/vtm#1063] [Q&A] [answered]
Currently I can't switch theme and need to use a specific one for different (online) tile providers: e.g. need to use `VtmThemes.MAPZEN` with `NextzenGeojsonTileSource` because it doesn't render with any other. How to use default themes properly, if it's possible with such providers like Nextzen, Mapilion etc.?

**Comment by devemux86** [comment, marked as answer]
Each map provider seems to change / restructure the OpenStreetMap tags, so they need different themes.
See the various samples about what theme each map source needs.

**Reply by Psijic** [reply]
Yes, I found the issue while inspected vtm-parent samples.
